user_sync/connector/directory_okta.py

```diff
@@ -58,42 +58,40 @@ class OktaDirectoryConnector(object):
     name = 'okta'
 
     def __init__(self, caller_options):
-        caller_config = user_sync.config.DictConfig('"%s options"' % OktaDirectoryConnector.name, caller_options)
+        caller_config = user_sync.config.DictConfig('%s configuration' % self.name, caller_options)
         builder = user_sync.config.OptionsBuilder(caller_config)
         builder.set_string_value('group_filter_format',
                                  '{group}')
         builder.set_string_value('all_users_filter',
-                                 'status eq "ACTIVE"')
-        # builder.set_string_value('user_email_format', '{email}')
+                                 'user.status == "ACTIVE"')
         builder.set_string_value('user_identity_type', None)
-        builder.set_string_value('logger_name', 'connector.' + OktaDirectoryConnector.name)
-        builder.set_dict_value('source_filters', {})
-
-        okta_url = builder.require_string_value('okta_url')
-        if 'https' not in okta_url:
-            okta_url = "https://" + okta_url
+        builder.set_string_value('logger_name', self.name)
+        host = builder.require_string_value('okta_url')
```

Comment:
Ha!  I meant to change the *configuration setting name* to `host` rather than `okta_url` (as well as the variable name).  Well, I got half of what I wanted.  Is there a reason you are not just calling this `host` since it doesn't have to be a URL?